spec/fieldhand/options_spec.rb

```diff
@@ -69,5 +69,25 @@ module Fieldhand
         expect(options.bearer_token).to eq('decafbad')
       end
     end
+
+    describe '#header' do
+      it 'defaults to empty hash' do
+        options = described_class.new({})
+
+        expect(options.headers).to be_empty
+      end
+
+      it 'can be overridden by passing headers in an option' do
+        options = described_class.new(:headers => { 'Crossref-Plus-API-Token' => 'decafbad' })
+
+        expect(options.headers).to eq({ 'Crossref-Plus-API-Token' => 'decafbad' })
+      end
+
+      it 'builds custom headers with bearer_token' do
```

Comment:
What is this test showing? How `bearer_token` and `headers` interact? If so, should we show how `bearer_token` _overrides_ any existing `Authorization` headers?

Reply:
> How bearer_token and headers interact?

Yes, exactly. 
I can see your point. It could be more useful. Will update it